src/main/kotlin/model/Delius.kt

```diff
@@ -40,12 +41,31 @@ class Delius private constructor() {
         ec2.add(this)
       }
 
+      system.addContainer(
+        "nDelius application",
+        "Application logic for Delius", "Java"
+      ).apply {
+        url = "https://github.com/ministryofjustice/delius"
+        uses(database, "connects to", "JDBC")
+        ec2.add(this)
+      }
+
       val elasticSearchStore = system.addContainer(
         "ElasticSearch store",
         "Data store for Delius content", "ElasticSearch"
       ).apply {
         Tags.DATABASE.addTo(this)
-        elasticSearchDeployment.add(this)
+        CloudPlatform.elasticsearch.add(this)
+      }
+
+      val deliusApi = system.addContainer(
+        "Delius API",
+        "API over the nDelius DB used by HMPPS Digital team applications and services", "Kotlin"
+      ).apply {
+        url = "https://github.com/ministryofjustice/delius-api"
+        uses(database, "connects to", "JDBC")
+        uses(HMPPSAuth.system, "Authenticates using")
```

Comment:
The use case for `defineRelationships` was intended to link to "other" systems and containers to remove the need for building a dependency graph when calling `defineModelEntities`

(Perhaps it might be better to name it `defineRelationshipsToOtherSystems` or something shorter 🤔)

If these `HMPPSAuth` references are moved there, the `DEPENDENCY_MODEL_ITEMS` won't be needed

Reply:
Yes, much better